khmer/utils.py

```diff
@@ -249,5 +249,25 @@ def write_record_pair(read1, read2, fileobj):
     write_record(read2, fileobj)
 
 
+class ReadBundle(object):
+    def __init__(self, *raw_records):
+        self.reads = [i for i in raw_records if i]
+        self.cleaned_reads = [r.sequence.replace('N', 'A') for r in self.reads]
+
+    def coverages(self, graph):
+        return [graph.get_median_count(r)[0] for r in self.cleaned_reads]
+
+    def both(self):
+        return zip(self.reads, self.cleaned_reads)
+
+    @property
+    def num_reads(self):
+        return len(self.cleaned_reads)
+
+    @property
+    def agg_length(self):
```

Comment:
we don't use 'agg' (presumably for 'aggregate') anywhere else in the code base - how about 'sum_length' or 'total_length'?